app/components/lists/route-plan-list/component.js

```diff
@@ -1,5 +1,9 @@
-import Ember from 'ember';
+import Ember from "ember";
+import computed from "ember-computed-decorators";
 
 export default Ember.Component.extend({
-  
+  @computed("model.@each.{hasRouteVisits}")
+  hasRouteVisitsModel(routePlans) {
```

Comment:
Let's rename this to: `validRoutePlans` to indicate that it is a collection and they are valid. I think in general we should avoid naming things with `fooModel` unless it is the `model` which is an ember convention. When creating a computed prop we can break out of that and actually name it something more informative. Also the term `has` would indicate that this is a boolean, not a collection.